task_sdk/src/airflow/sdk/definitions/connection.py

```diff
@@ -57,6 +58,15 @@ def get_uri(self): ...
 
     def get_hook(self): ...
 
+    @classmethod
+    def get(cls, conn_id: str) -> Any:
+        from airflow.sdk.execution_time.context import _get_connection
+
+        try:
+            return _get_connection(conn_id)
+        except Exception:
+            raise
```

Comment:
No point to the try/except if we just re-raise it.

```suggestion
        return _get_connection(conn_id)
```

Reply:
😓